tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -108,6 +111,7 @@ public VerifiableConsumer(KafkaConsumer<String, String> consumer,
         this.useAutoCommit = useAutoCommit;
         this.useAsyncCommit = useAsyncCommit;
         this.verbose = verbose;
+        this.log = (new LogContext("[VerifiableConsumer] ")).logger(VerifiableConsumer.class);
```

Comment:
You don't need the outer `(), do you?

Reply:
The log context is only useful when we have a way to distinguish between different instances of the same class. If all we get is the classname here, then we probably don't need it since the log message will typically already include that.